src/pages/registration/FileList.tsx

```diff
@@ -99,36 +95,56 @@ export const FileList = ({ title, files, uppy, remove, baseFieldName }: FileList
                     <Box sx={{ display: 'flex', gap: '0.5rem', alignItems: 'center' }}>
                       {t('registration.files_and_license.license')}
                       <HelperTextModal
+                        maxWidth="xl"
                         modalTitle={t('registration.files_and_license.licenses')}
                         modalDataTestId={dataTestId.registrationWizard.files.licenseModal}
                         buttonDataTestId={dataTestId.registrationWizard.files.licenseHelpButton}>
-                        <Typography sx={{ mb: '1rem' }}>
-                          {t('registration.files_and_license.file_and_license_info')}
-                        </Typography>
-                        {activeLicenses.map((license) => (
-                          <Box key={license.id} sx={{ mb: '1rem', whiteSpace: 'pre-wrap' }}>
+                        <Trans
+                          i18nKey="registration.files_and_license.file_and_license_info"
+                          components={{
+                            p: <Typography gutterBottom />,
+                            ccLink: <OpenInNewLink href="https://creativecommons.org/licenses/" />,
+                          }}
+                        />
+                        <Divider sx={{ my: '1rem', borderColor: 'black', borderWidth: '1px' }} />
+                        <Box
+                          sx={{
+                            display: 'grid',
+                            gridTemplateColumns: { md: 'repeat(1, 1fr)', lg: 'repeat(3, 1fr)' },
+                            gap: '1.5rem',
+                          }}>
+                          {activeLicenses.map((license) => (
+                            <Box key={license.id} sx={{ whiteSpace: 'pre-wrap' }}>
+                              <Box sx={{ display: 'flex', alignItems: 'center', gap: '0.5rem', mb: '0.5rem' }}>
+                                <Box component="img" src={license.logo} alt="" sx={{ width: '5rem' }} />
+                                <Typography component="h2" variant="h3" gutterBottom>
```

Comment:
`gutterBottom` her ser bare ut til å gjøre til at bilde og heading _ikke_ blir sentrert?
```suggestion
                                <Typography component="h2" variant="h3">
```

![bilde](https://github.com/user-attachments/assets/56ee9bee-ccb1-4b16-b743-c9e9f921eddd)